easybuild/easyconfigs/a/aria2/aria2-1.36.0-GCCcore-10.2.0.eb

```
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'aria2'
+version = '1.36.0'
+
+homepage = 'https://aria2.github.io'
+description = "aria2 is a lightweight multi-protocol & multi-source command-line download utility."
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/aria2/aria2/releases/download/release-%(version)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['b593b2fd382489909c96c62c6e180054c3332b950be3d73e0cb0d21ea8afb3c5']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Autotools', '20200321'),
+    ('CppUnit', '1.15.1'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libxml2', '2.9.10'),
+    ('SQLite', '3.33.0'),
+    ('c-ares', '1.17.2'),
+    ('OpenSSL', '1.1.1h'),
```

OpenSSL is a wrapper for the latest installed OpenSSL in your system.
```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
```